src/components/utils/generateSpace.ts

```diff
@@ -0,0 +1,29 @@
+import { ViewStyle } from 'react-native'
+
+import { SpacingValue } from '~components'
+import { space, scale } from '~constants/newTheme'
+
+export const generateSpace = (value?: SpacingValue, key?: keyof ViewStyle): ViewStyle | false => {
```

Comment:
Maybe we can add some comments to this function to tell what is this doing?

Showing some example for input and out put would be good.

It's for future maintenance, because right now I don't remeber whats going on here :D